elytron-oidc-client/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -206,6 +206,7 @@ elytron-oidc-client.secure-server.credential.algorithm=The credential signature
 elytron-oidc-client.secure-server.redirect-rewrite-rule.add=Add the rewrite rule for the redirect URI. The rewrite rule is an object notation, where the key is a regular expression with which the redirect URI is matched and the value is the replacement String. You can use the '$' character to use for back-reference in the replacement String.
 elytron-oidc-client.secure-server.redirect-rewrite-rule.remove=Remove the rewrite rule for the redirect URI.
 elytron-oidc-client.secure-server.redirect-rewrite-rule.replacement=The replacement String in the rewrite rule for the redirect URI.
+elytron-oidc-client.secure-server.scope=A space separated list of scope values that should be used in the OIDC authentication request. Scopes can be used to request that specific sets of information be made available as Claim Values.
```

Comment:
@PrarthonaPaul Have you done any manual tests with `secure-server` configuration with `scope` configured? We should probably create an issue for adding a test case for this to the HAL testsuite (https://github.com/hal/berg). That's where other `secure-server` related tests currently are.

Reply:
I have done manual tests and it works. 
I have added the issue for the HAL tests here: https://github.com/hal/berg/issues/118